docs/satellite-clone.md

```diff
@@ -13,6 +13,12 @@
 #### Prerequisites ####
 
 1. You will need files from a katello-backup (`katello-backup` on the `Satellite server`).
+   Required backup files:
```

Comment:
@sghai the backup can also be from a 6.1 machine, you bring up a good point in that we don't actually even handle the elastic_data.tar.gz file in sat-clone. The data will be recreated on a reindex, but we should restore the backup file if its there. I'll open an issue and we can handle it separately